### DISCUSSION TITLE
VueUiCarouselTable component configuration

### DISCUSSION THREAD

**Discussion by llds66** [graphieros/vue-data-ui#75] [Q&A] [answered]
Why does the VueUiCarouselTable setting dataset `head` appear on the left instead of the top of the demo? What do I need to set up? I didn't find the option
`document`
![image](https://github.com/user-attachments/assets/7a7873a8-7506-494d-ba84-9b888fa9c374)
`mine`
![image](https://github.com/user-attachments/assets/54dff9a5-4700-4a4b-b064-f99b785ae9c7)

**Comment by graphieros** [comment, marked as answer]
This is the responsive layout, which kicks in under 400px by default.
If you never want responsive mode you can set a very low responsiveBreakpoint value in the config.

**Reply by llds66** [reply]
OK, thanks
